listenbrainz/listenstore/listenstore.py

```diff
@@ -1,70 +0,0 @@
-# coding=utf-8
-
-import calendar
-import logging
-import time
-
-from listenbrainz.listenstore import ORDER_ASC, ORDER_DESC, DEFAULT_LISTENS_PER_FETCH
-
-
-class ListenStore(object):
-    MAX_FETCH = 5000          # max batch size to fetch from the db
-    MAX_FUTURE_SECONDS = 600  # 10 mins in future - max fwd clock skew
-
-
-    def __init__(self, logger):
```

Comment:
The `self.log` parameter is used at a few places by both RedisListenStore and TimescaleListenStore. For now, I have added an __init__ method accepting logger to both but IIRC the intent is to move away from using `current_app` logger everywhere in which the `self.log` will also go away.